Makefile

```diff
@@ -0,0 +1,7 @@
+build:
+	pipenv install
+	pipenv install --dev
```

Comment:
Do you need both lines? Won't pipenv install --dev also install any usual requirements?

Reply:
Removed the duplicate line